client/src/components/AdminContainer/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Layout, Button } from 'antd';
+import logo from '../../assets/images/logo.png';
+import './style.css';
+
+const { Header } = Layout;
+
+const AdminContainer = ({ children, buttonContent, buttonFunction }) => {
+  return (
+    <>
+      <Layout>
+        <Header>
+          <div className="admin-header-image">
+            <img src={logo} alt="logo" />
+          </div>
+          <div className="admin-header-btn">
+            {buttonContent !== undefined && (
+              <Button onClick={buttonFunction} type="primary" danger>
+                {buttonContent}
+              </Button>
+            )}
+          </div>
+        </Header>
+        <div className="admin-container">
+          <div className="admin-container-side">
+            <div className="admin-side">
+              <ul className="admin-menu">
+                <li>
+                  <NavLink
+                    exact
+                    to="/admin"
+                    className="admin-menu-a"
+                    activeClassName="admin-menu-active"
```

Comment:
@tomduggan85 I come back to use NavLink instead of Link, this what I was meant
without exact when I go to /admin/cohort will make hover on two elements (Home, Cohorts) like this
![604](https://user-images.githubusercontent.com/55782435/78770615-7b11ad00-7997-11ea-9608-06fe6d2b053c.png)
so  I use **exact** here

Reply:
Ah, that makes sense.  Cool!